command/commands/team.py

```diff
@@ -2,16 +2,21 @@
 import argparse
 import logging
 import shlex
+from flask import jsonify
 from interface.github import GithubAPIException
 from model.team import Team
+from model.user import User
+from command.util import check_credentials
 
 
 class TeamCommand:
     """Represent Team Command Parser."""
 
     command_name = "team"
     desc = "for dealing with " + command_name + "s"
-    permission_error = "Insufficient permission level to run command!"
+    permission_error = "You do not have the sufficient " \
+                       "permission level for this command!"
```

Comment:
I think it would flow better if this was changed to `"You do not have sufficient permissions for this command!"`